content/en/dashboards/ddsql_editor/reference/data_types.md

```diff
@@ -0,0 +1,64 @@
+---
+title: DDSQL Data Types
+---
+
+{{< callout url="https://google.com">}}
+DDSQL is in private beta.
+{{< /callout >}}
+
+## Data types
+
+DDSQL implements a simplified version of the SQL type system that is mostly descended from SQLite.
+
+### Base types
+
+| SQL name   | Aliases                  | Description |
+|------------|--------------------------|-------------|
+| integer    | int                      | Storage is always int64. |
+| text       | varchar, json            | Storage is always unlimited-length UTF-8. |
+| real       | double                   | Storage is always IEEE-754 float64. |
+| timestamp  | timestamp without time zone | SQL standard datetime type. |
+| interval   |      | Encodes a span of time. Useful for setting the default interval environment variable with `SET`. |
```

Comment:
could you remove all mentions of `interval` in this page? it's not yet supported on the execution path we'll be using.